File: ExcelSheetColumnTitle/ExcelSheetColumnTitle.js
/**
 * @param {number} columnNumber
 * @return {string}
 */
var convertToTitle = function (columnNumber) {
  let res = "";
  while (columnNumber > 0) {
    let r = columnNumber % 26;
    let d = parseInt(columnNumber / 26);
    if (r == 0) {
      r = 26;
      d -= 1;
    }
    res += String.fromCharCode(64 + r);
    columnNumber = d;
  }
  return res.split("").reverse().join("");
};

File: ExcelSheetColumnTitle/ExcelSheetColumnTitle.py
class Solution:
    def convertToTitle(self, columnNumber: int) -> str:
        arr = []
        while columnNumber > 0:
            arr.append(chr((columnNumber - 1) % 26 + ord("A")))
            columnNumber = (columnNumber - 1) // 26
        arr.reverse()
        return "".join(arr)


class Solution:
    def convertToTitle(self, columnNumber: int) -> str:
        s = "" 
        while columnNumber > 0:
            columnNumber,rem = divmod(columnNumber-1,26)
            s += chr(65+rem)
        return s[::-1]